src/plugins/widgets/bitcoin/Bitcoin.sass

```diff
@@ -0,0 +1,66 @@
+.Bitcoin
+    display: flex
+    justify-content: center
+    grid-gap: 2rem
+    // padding to make space for pseudo elements
+    padding-top: 20px
+    padding-left: 20px
+    position: relative
+
+    .bitcoin-block
+        background: repeating-linear-gradient(rgb(45, 51, 72), rgb(45, 51, 72) 0.005575%, rgb(147, 57, 244) 0.005575%, rgb(16, 95, 176) 100%)
+        cursor: pointer
+        width: 125px
+        height: 125px
```

Comment:
If you specify the block sizes in `em` units, the blocks will automatically scale with the widget's build in size control.